xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
Yes, that would be better in deed. What about the users that want to save "decoding" power and therefore only want to decode 5.1 dts instead of the much more demanding dts-hd? Especially when output is 2 channel only?

Think of the TV 2.0 watching users only ... no need at all to decode 7.1 dts-hd and then downmix to 2.0 channels ... default is disabled btw. ... to care for those user automatically.

https://github.com/fritsch/xbmc/commit/e2a3350a6108d344ffe49819d7de6d0ec1163d97?w=0